server/lib/mod-utils.js

```diff
@@ -970,6 +970,10 @@
       return 'Excusal granted (' + description.toLowerCase() + ')';
     }
 
+    if (status === 'Deferred') {
+      return 'Deferral granted (' + description.toLowerCase() + ')';
```

Comment:
will the description be available here if it is deferred?

Reply:
yup, its using the same excusal code field as excusals